codegen-sbt/src/sbt-test/compiletime-codegen/simple/project/Libraries.scala

```diff
@@ -0,0 +1,21 @@
+import sbt._
+
+object Libraries {
+
+  val zioVersion     = "1.0.11"
+  val calibanVersion = "1.1.1"
```

Comment:
Is there a way to depend on the current caliban instead?

Reply:
Yep. I changed the code.